L1Trigger/L1TTrackerDTC/plugins/L1TTrackerDTCProducer.h

```diff
@@ -0,0 +1,40 @@
+#ifndef __L1TTrackerDTC_L1TTRACKERDTCPRODUCER_H__
```

Comment:
plugins are recommended not to have separate header files

Reply:
I never saw an EDProducer without header file. Can you point me to an example?

Reply:
https://github.com/cms-sw/cmssw/blob/master/RecoLocalCalo/HGCalRecProducers/plugins/HGCalRecHitProducer.cc

The class is declared and implemented in the cc file (since, as a plugin, it is not intended to be included in any other source files)